website/content/docs/k8s/annotations-and-labels.mdx

```diff
@@ -105,6 +105,13 @@ The following Kubernetes resource annotations could be used on a pod to control
           "consul.hashicorp.com/connect-service-upstreams":"[service-name].svc.[service-dc].dc:[port]"
         ```
 
+      - Sameness group: Place the sameness group name after the service name followed by `.sg` to specify a sameness group for an upstream service. When a sameness group CRD has `spec.defaultForFailover` set to `true`, this field is not required. Refer to [sameness group configuration entry reference](/consul/docs/connect/config-entries/sameness-group) for more information
```

Comment:
Similar to my other `destination_sameness_group` comment, I don't believe a sameness-group can be explicitly targeted as an upstream. We currently only support configuring targeting SGs for failover via resolvers.